addons/payment/data/payment_provider_data.xml

```diff
@@ -302,6 +302,7 @@
                          ref('payment.payment_method_upi'),
                          ref('payment.payment_method_wallets_india'),
                          ref('payment.payment_method_paylater_india'),
+                         ref('payment.payment_method_emi'),
```

Comment:
I think it would be better to add post fix `_india` as this method is only supported in India (by updating PM and PM code)
[See comment](https://github.com/odoo/odoo/pull/164112#discussion_r1590747803) 

This way we can be sure this is only India related PM and if other country support EMI then can add EMI PM for their country and does not get confused ;)

Reply:
@AntoineVDV can I have your opinion about this?

Reply:
It depends on what "EMI" stands for:
- If it's a generic name like "wallet" or "voucher" then, yes, it should be postfixed with `_index` to distinguish it from, say, `emi_europe`.
- If it's a brand name like "UPI" or "PayPal" then, no, for the same reason as UPI: there will never be a `upi_brazil` PM.

Reply:
![image](https://github.com/odoo/odoo/assets/77771734/7108ab8b-532e-4e20-a977-5f9de5def269)
It seems generic and I think other countries also use it :see_no_evil: